dev/dll/SharedHelpers.cpp

```diff
@@ -198,6 +198,12 @@ bool SharedHelpers::IsIsLoadedAvailable()
     return s_isAvailable;
 }
 
+bool SharedHelpers::IsCompositionRadialGradientBrushAvailable()
```

Comment:
>IsCompositionRadialGradientBrushAvailable [](start = 20, length = 41)

Is there an interface we can QI for instead? platform look ups are expensive and should be avoided if possible.

Reply:
Do we do that anywhere else? 
I was trying to follow the other SharedHelpers checks which also use ApiInformation, and I think this lookup in windows.foundation.metadata should only do the lookup once on first use of the brush per app launch.